unlock-app/src/__tests__/utils/checkoutLockUtils.test.ts

```diff
@@ -200,17 +200,15 @@ describe('Checkout Lock Utils', () => {
 
       const lock = {
         fiatPricing: {
-          usd: {
-            keyPrice: '7.4',
-          },
+          amount: 20.22,
```

Comment:
I think it is important qe keep track of the currency there... and allow us to eventually support multiple ones

Reply:
added details for `usd` currency in the hook